MovieEnthusiast.Domain/Entities/Movie.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MovieEnthusiast.Domain.Entities;
+
+public class Movie
+{
+    public int Id { get; set; }
+
+    public string Title { get; set; } = string.Empty;
```

Comment:
`public required string Title { get; set; }`
[ref](https://learn.microsoft.com/en-us/ef/core/miscellaneous/nullable-reference-types#non-nullable-properties-and-initialization)